custom_conf.py

```diff
@@ -212,6 +212,146 @@
 # that you want to link to:
 # manpages_url = "https://manpages.ubuntu.com/manpages/noble/en/man{section}/{page}.{section}.html"
 
+############################################################
```

Comment:
Is there anything in here that we expect users to change/customise?

All the defaults should go into `conf.py`, and `custom_conf.py` should only have things that we allow users to customise.

Reply:
Yes. This is included here with the expectation that there may be things users want to customise. For instance, the Brand Store team adds subtitles and other information to the title page.

Of these elements, considering which ones I do and do not customise in my use case - only the `latex_engine`, `latex_show_pagerefs` and `latex_show_urls` would remain uncustomised.

Reply:
I don't think we should allow too much customisation. I understand we might need some, but we should have a recommended setup that covers what most teams want to do.
So for additional subtitles and other information, those should be included in the standard setup (and can then be left empty if they aren't needed). 

As discussed offline, having the full preamble in the `custom_conf.py` file will be confusing/scary to most users. And it should only be customised by users that actually know what they are doing.
How about putting the content of `latex_elements` into a file (in `.sphinx`) and reading in that file in `conf.py`. That would make it possible for users to modify the file, but it wouldn't make everybody wonder if they need to do something there.
We could even make it possible to customise the file name (have a default file name in `conf.py`, unless another one is specified in `custom_conf.py`), so users can maintain their own file and don't have to modify the one we provide.

Similar with the other options - it should work without any customisation, but if a team needs different values, `conf.py` could have logic to take custom options defined in `custom_conf.py` into account.

Reply:
I've moved this to a separate file in `.sphinx`.